src/util.h

```diff
@@ -175,6 +175,15 @@ void AllocateFileRange(FILE *file, unsigned int offset, unsigned int length);
 bool RenameOver(fs::path src, fs::path dest);
 bool LockDirectory(const fs::path& directory, const std::string lockfile_name, bool probe_only=false);
 
+//! Return the original FILE* unchanged. On POSIX systems that support it,
+//! also advise the kernel that the file will be accessed sequentially.
```

Comment:
Might want to mention also that this advises `WILLNEED` in addition to just `SEQUENTIAL`.